frontend/packages/operator-lifecycle-manager/src/components/descriptors/reference/reference.md

```diff
@@ -742,10 +690,44 @@ The Dependent Field(s) will be displayed only when the current value of the Cont
   </tr>
 </table>
 
+
+#### 21. hidden
+
+**x-descriptors**
+
+This descriptor allows you to exclude non-user facing fields from the &quot;auto-generated creation form&quot; by specifying them as &quot;hidden&quot; schema fields. See example for Portworx Operator:
+
+```yaml
+…
+ - displayName: HIDDEN FIELDS - Network and all its children fields
+   description: >-
+      HIDDEN FIELDS - The network related fields to be used by Portworx for data traffic.
+   path: network
+   x-descriptors:
+     - 'urn:alm:descriptor:com.tectonic.ui:hidden'
+ - displayName: HIDDEN FIELDS - Start Port
+   description: >-
+      HIDDEN FIELDS - It is the starting port in the range of ports used by Portworx.
+   path: startPort
+   x-descriptors:
+     - 'urn:alm:descriptor:com.tectonic.ui:hidden'
+…
+```
+
+**Usage**
+1. You can apply &quot;hidden&quot; to any schema property path (doesn't have to be a &quot;leaf&quot; in the schema). All the children properties, if any, will be hidden as well.
+2. The descriptor would hide that property and all its children fields.
+3. The field(s) would be hidden on the UI but still can be access through the YAML editor.
+
+---------------------------------------
+
+
 ## statusDescriptors
-A statusDescriptor is for describing the property of a `status` field in your Custom Resource. Each statusDescriptor provides a UI widget for exposing the CR instance on the DISPLAY view in the OpenShift Console.
+A statusDescriptor is for describing the property of a `status` field in your Custom Resource. The structure of the statusDescriptors is similar, including the same fields you need to specify. The only difference is the set of valid `x-descriptors`, which can be found below.
```

Comment:
```suggestion
A statusDescriptor is for describing the `status` stanza in your Custom Resource. The structure of the statusDescriptors is similar to specDescriptors and includes the same fields. The only difference is the set of valid `x-descriptors` described below.
```

Reply:
Since the `status` **stanza** means the full `status` block in the manifest, let's keep that sentence as-is and go with the rest of your suggestions, thanks Jon!